src/Controller/ArticlesController.php

```diff
@@ -492,7 +492,9 @@ private function createFirstPage(string $id, array $arguments) : Response
                 return $body;
             });
 
-        return new Response($this->get('templating')->render('::article-text.html.twig', $arguments));
+        $arguments['google_scholar_metadata'] = (bool) $request->headers->get('X-eLife-Google-Scholar-Metadata', false);
```

Comment:
Preferred to use a security voter (`RequestHeaderVoter`), but not sure how to sanely add the `Vary`. (Unless we don't bother and rely on the CDN.)

Reply:
This looks like a sane way as an article-specific choice that is kept in the `ArticlesController`